adapters/rubicon/rubicon.go

```diff
@@ -849,19 +850,30 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *ada
 			thisImp.Video = nil
 		}
 
-		siteExt := rubiconSiteExt{RP: rubiconSiteExtRP{SiteID: rubiconExt.SiteId}}
 		pubExt := rubiconPubExt{RP: rubiconPubExtRP{AccountID: rubiconExt.AccountId}}
 
 		if request.Site != nil {
 			siteCopy := *request.Site
-			siteCopy.Ext, err = json.Marshal(&siteExt)
+			siteExtRP := rubiconSiteExt{RP: rubiconSiteExtRP{SiteID: rubiconExt.SiteId}}
+			if err := updateSiteExtWithIabAttribute(&siteExtRP, siteCopy.Content.Data); err != nil {
+				errs = append(errs, err)
+				continue
+			}
+
+			siteCopy.Ext, err = json.Marshal(&siteExtRP)
+			if err != nil {
+				errs = append(errs, err)
+				continue
+			}
+
 			siteCopy.Publisher = &openrtb2.Publisher{}
 			siteCopy.Publisher.Ext, err = json.Marshal(&pubExt)
 			rubiconRequest.Site = &siteCopy
 		}
+
 		if request.App != nil {
```

Comment:
Nitpick: We don't need the second check because Prebid Server core filters requests so they either have a non-`nil` `Site` field or non-nil `App` field. Therefore, it'd be safe to use an `else` clause instead if you want to.

Reply:
Done. Nice catch, thanks)